CHANGELOG.md

```diff
@@ -41,6 +26,10 @@
   * Deprecate `firstNonNull`. Users of this function should migrate to
     `var v = o1 ?? o2 ?? o3 ?? o4; ArgumentError.checkNotNull(v);`. This will be
     removed in 4.0.0
+  * Eliminate dependency on package:meta.
+  * Fix: `TreeSet.last` was incorrectly returning the minimum node (same
+    value as `TreeSet.first`). It now correctly returns the maximum
+    node.
```

Comment:
This last bug never existed in 2.1.5 -- it was introduced in 3.0.0-nullsafety and fixed prior to 3.0.0.

Reply:
ack!